internal/services/netapp/netapp_volume_resource.go

```diff
@@ -614,6 +658,8 @@ func resourceNetAppVolumeRead(d *pluginsdk.ResourceData, meta interface{}) error
 		d.Set("service_level", string(pointer.From(props.ServiceLevel)))
 		d.Set("subnet_id", props.SubnetId)
 		d.Set("network_features", string(pointer.From(props.NetworkFeatures)))
+		d.Set("smb_non_browsable_enabled", string(pointer.From(props.SmbNonBrowsable)))
+		d.Set("smb_access_based_enumeration_enabled", string(pointer.From(props.SmbAccessBasedEnumeration)))
```

Comment:
those are not being converted to boolean, handling is needed here since what comes from Azure is "Enabled"/"Disabled" strings.

Reply:
fixed